### DISCUSSION TITLE
Support forum links to toxic anti-social media site “reddit”, leaving sane humans with no support.

### DISCUSSION THREAD

**Discussion by navid-zamani** [uBlockOrigin/uBlock-issues#2493] [General] [answered]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

**See below**
[Since there is no better place, and it is a bug (that is actually harmful), this goes here.]

### A specific URL where the issue occurs.

https://github.com/uBlockOrigin/uBlock-issues/issues/new/choose

### Steps to Reproduce

When filing new issues for uBO,
there is a category for support questions. (Which is generally a good idea.)
Click (or hover over) that button.

### Expected behavior

It should stay on Github, since it is already perfectly suited for support, as that is what a ticketing system is for.  

The issue would be rather, that support questions drown the actual bugs, leading to it becoming useless for tracking those.

But this is rather easily solved, by merely keeping them 100% separate (if necessary in a separate repository), while still staying on Github.  

### Actual behavior

The button goes to an external malware site of the anti-social media category. Those are sites that are deliberately designed to harm people by making them addicted and not able to leave (like all such sites) by pushing emotional buttons (like getting them angry/scared), and usually, without this add-on, would make money by feeding victims hostile manipulative input, designed to get them to make inferior financial decisions (aka “advertisement”).

**This leads to direct bodily harm (mostly in the form of depression, anxiety and loneliness) as well as financial harm.**  
It is another form of the very thing this add-on was created to stop.

Which is why such sites are generally considered social malware and a big no-no among individual people. (So pretty much everyone outside of the corporate and political “swarm lifeforms”.)  
It is also why people are generally blocking such sites here. At work and at some (esp. with children). (And sometimes recommend talking to a mental health counsellor.)

### uBO version

1.46.0

### Browser name and version

Mozilla Firefox 109.0.1

### Operating System and version

Gentoo Linux

**Comment by mapx-** [comment, marked as answer]
For support questions you can open issues on the `discussions` threads:

filters: https://github.com/uBlockOrigin/uAssets/discussions
questions about the extension itself: https://github.com/uBlockOrigin/uBlock-issues/discussions

**Comment by uBlock-user** [comment]
Converted to Discussions.
